msteams-platform/apps-in-teams-meetings/meeting-apps-apis.md

```diff
@@ -1524,6 +1528,161 @@ The following code provides an example of meeting end event payload:
 | **value.EndTime** | The meeting end time in UTC. |
 | **locale**| The locale of the message set by the client. |
 
+### Receive meeting participant events
+
+Your bot can receive real-time meeting events such as participant join and leave events through the meeting event subscription in Developer Portal. A bot receives the participant events only if subscribed to these events in Developer Portal.
```

Comment:
Duplicate statements. Only keep one of the 2 statements. 
Preferably something like:  A bot receives the participant events only if subscribed to these events in Developer Portal.

Reply:
Updated

Reply:
I don't see any change related to this. What I'm saying here is that in my opinion both statements have the same meaning and there is no need to keep both. 
"A bot can receive the participant events only if subscribed to these events in Developer Portal."

Reply:
Updated